assets/src/edit-story/app/story/useStoryReducer/reducers/setBackgroundElement.js

```diff
@@ -55,11 +54,12 @@ function setBackgroundElement(state, { elementId }) {
 
     const defaultBackgroundElement = page.defaultBackgroundElement;
 
-    // Unset isBackground for the element, too.
+    // Unset isBackground and backgroundOverlay for the element, too.
```

Comment:
Is it confirmed that the overlay should not be stored for the media at all? With this logic, when the user clicks "Remove as background" and then "Set as background" right after, the overlay is gone. Is that the expected behavior?

Reply:
Sorry, forgot to reply here, but yes that is the expected behaviour. It would be weird of the overlay stuck around while not visible.